### ISSUE TITLE
Privacy Policy

### ISSUE THREAD

**Issue by calendee** [ultrahorizon/UH-VPN-Docs#8]
Hello!  UH VPN looks pretty interesting.

**Is your feature request related to a problem? Please describe.**
No problems.

**Describe the solution you'd like**
I'd love to see some privacy policies and terms of service.

- Does a VPN hosted on a user's server ever send data to Ultra Horizon?
- What information is passed by the server to UH?
- Are there any logs recorded on the server or passed to UH?
- Basically, please explain *everything* people need to know as to whether or not to trust this service.

**Comment by calendee** [comment]
Oops!  Looks like this is a duplicate of #6.  However, those links don't appear to be accessible from the main site or the documentation page.  It would be great to add them there.

**Comment by AnthonyWharton** [comment]
Hi @calendee,

Thanks for the interest, great to see new faces!  You are indeed correct, the links aren't currently accessible from the main docs page, however I believe the UH VPN Apps should all provide direct links to it upon first launch.

In any case the link is here: https://docs.uh-vpn.com/en/latest/privacy.html

There were some internal review of documents which is why they went offline originally - we'll make sure that these links become visible ASAP (likely under another "Legal" section on the left navigation bar, or under "Support").

Let me know if there's anything else we can do, thanks!

**Comment by AnthonyWharton** [comment]
Oh, and for clarity's sake if you are picking through it, I should add in the current revision of the Privacy Policy document we refer to:
 - "Groups" as "Organisations"
 - "Persons" as "Identities"
 - "Devices" as "Profiles"

There was a nomenclature change before we released UH VPN to the public for clarify, and this is one of the last few places that it hasn't been changed yet! We'll get that sorted ASAP!

**Comment by jwsi** [comment]
Privacy policy updated in changeset 739904db040daec17d7a55e472759ee6b1dfdc18
